src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,53 +1,109 @@
 package core.basesyntax;
 
-/**
- * <p>Реалізувати свій ArrayList який імплементує інтерфейс List. Дотриматися основних вимог щодо
- * реалізації ArrayList (default capacity, newCapacity...)</p>
- */
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private T[] elementData;
+    private int size;
+    private int capacity;
```

Comment:
you can always get capacity from elementData

Reply:
fixed